src/npm-on-ipfs/index.js

```diff
@@ -30,27 +29,17 @@ export default function (ctx) {
   let opt = store.get(CONFIG_KEY, null)
   const exists = isPkgInstalled()
 
-  // Confirms if the package is still (un)installed because the user
-  // might change it manually.
-  if (opt !== null || exists !== false) {
+  if (opt === null) {
```

Comment:
Is `exists` used anymore?

Can you leave the comment in, or correct to describe current behavior?

Reply:
Yes, `exists` is used, right on the next two lines. I did not write a comment because I thought the info provided by `logger.info` would work as a comment. If I add one, I will basically just copy the next line